src/views/product.js

```diff
@@ -17,6 +17,7 @@ export default class ProductView extends View {
    * add event listener which triggers resize when the product image is loaded.
    */
   resizeOnLoad() {
+    if (!(this.component.config.product.contents.img || this.component.config.product.contents.imgWithCarousel)) { return; }
```

Comment:
I feel like there should be a linter rule for line length 😄 

This is pretty long.